### ISSUE TITLE
Incosistant hover effetc in Calc 

### ISSUE THREAD

**Issue by Darshan-upadhyay1110** [CollaboraOnline/online#6582]
**To Reproduce**
1. Go to Calc ( .ods doc)
2. Hover over the column header or row headers 
3. You can see the unnatural behaviour .

**Expected behavior**
Hover effect shoul be back to it's default state when cursor goes out of the box of the header column.
 
**Actual behavior**
Current beahviour is like it was not backporting to default state when mouse move out to the column box

**Screenshots**

[Screencast from 09-06-23 03:55:42 PM IST.webm](https://github.com/CollaboraOnline/online/assets/61383886/d4a86365-facf-4610-add3-b93435ec7361)

**Additional context**
To see better about this issue please reproduce this in dark mode. will get better idea on that.

**Comment by thebearon** [comment]
Indeed, I have noticed this as well. It might be a regression from a while ago.

**Comment by Tex2002ans** [comment]
I confirm this is still an issue in:

> COOLWSD version: 23.05.5.1 (git hash: d5cda355)
> LOKit version: Collabora Office 23.05.5.1 (git hash: 58c02b4)
> Served by: Linux Mint 21.1
> Server ID: 4c3986e9

**Comment by smehrbrodt** [comment]
No longer an issue.
